### DISCUSSION TITLE
دوره‌همی دنیای برنامه‌نویسی، ۱-۵-۱۴۰۱

### DISCUSSION THREAD

**Discussion by alimd** [AliMD/alwatr#173] [Webinar & Meetups]
# دوره‌همی دنیای برنامه‌نویسی

`در تاریخ: ۱-۵-۱۴۰۱`

### سلام و عرض ادب، خیلی خوش‌آمدید

لطفا اگر مطلب مفیدی در رستای محتوای مطرح شده دارید به اشتراک بزارید و یا به یادگار پستی بزارید 🌹 

[ویدیو دوره‌همی در آپارات](https://www.aparat.com/v/a9h0M)

**مباحت مطرح شده:**

۱. مقاله‌ای از Peter Norvig مدیر تحقیقات و یکی از برنامه‌نویسان ارشد گوگل درباره جملات فریبنده‌ای همچون آموزش برنامه‌نویسی در ۲۴ ساعت.
[لینک مقاله](https://www.udacity.com/blog/2014/01/peter-norvig-teach-yourself-programming.html)

۲. اهمیت Github و معرفی پروژه [Alwatr](https://github.com/AliMD/alwatr)

۳. چرا یادگیری [حل مکعب روبیک](https://rubiks.com/solve-it) برای شما واجبه!
و ۴ نکته‌ی آموزشی مهم که از حل مکعب روبیک باید یادگرفت

[مطالب بیشتر در کانال تلگرام](https://t.me/AliMDSchool)

**Comment by alimd** [comment]
https://www.kaggle.com/

**Reply by Masterchoir** [reply]
Done!

**Reply by alimd** [reply]
لینک بده همو فالو کنیم 🤣 
https://www.kaggle.com/alidotmd

**Comment by LPLNKazemi** [comment]
سلام و عرض ادب و احترام.
ان شاالله این سلسله جلسات ادامه داشته باشه.
ممنون ومتشکر

**Comment by A-sina** [comment]
salam o dorood
besiar khoob bood va lezat bordim 
mamnoon az shoma

**Comment by SMHosseini77** [comment]
سلام و خسته نباشید
جلسه خوبی بود
استفاده کردیم
ممنونم

**Comment by taha-agh** [comment]
سلام

**Comment by FatemeGharavian** [comment]
سلام استاد 
متشکرم از آموزش عالی و خوبتون

**Comment by shosseinfarjadi** [comment]
file:///C:/Users/ABIT/Downloads/jsbin.xoxaxuxoyu.2%20(2).html
لینک بازی
